src/redux/coinlist.js

```diff
@@ -0,0 +1,90 @@
+/* @flow */
+import type {
+  Dispatch as ReduxDispatch,
+} from 'redux';
+import * as _ from 'lodash';
+
+import type { CoinList } from '../Types';
```

Comment:
Yeap removing it works too, but I saw another code example from Redux use it and I agree that it mostly just makes it clear.  Another thought here is I wonder if we should have a naming convention for types, because `CoinList` can totally also be the name of a component.  At my company we might call it `ICoinList` or something like that (but we use typescript)